cymetric/convenient_interface.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python
+
+from pyne import nucname
+import pandas as pd
+import numpy as np
+import cymetric as cym
+import warnings
```

Comment:
As per PEP group these according to Python standard library imports, 3rd party library imports, and the local package imports.